src/pages/AuthPage.tsx

```diff
@@ -23,17 +26,32 @@ export const AuthPage = () => {
   return (
     <IonPage>
       <IonContent>
-        <form
+
+      <div className="log">
+          
+
+          {/* <IonIcon name="logo-javascript"></IonIcon> */}
+          {/* <IonIcon icon={logoJavascript} size="large"></IonIcon> */}
+          
+              <div className="logoIMG">
+                <img src="/public/assets/ui/logo.svg"  />
+                <h1 className="logotitle ion-margin-top">ovoflow</h1>
+              </div>
+
+          <div className='buttons'>
+            <div className="funcions ion-padding-horizontal">
+
+            <form
           className="login-form ion-padding"
           onSubmit={handleSubmit(onSubmit)}
         >
           <IonItem fill="outline">
-            <IonLabel position="floating">Username</IonLabel>
-            <IonInput {...register("username")} required></IonInput>
+            {/* <IonLabel position="floating">Username</IonLabel> */}
+            <IonInput placeholder="username" {...register("username")} required></IonInput>
           </IonItem>
           <IonItem fill="outline">
-            <IonLabel position="floating">Password</IonLabel>
-            <IonInput
+            {/* <IonLabel position="floating">Password</IonLabel> */}
+            <IonInput placeholder="password"
```

Comment:
Could you walk me through the decision to use a `placeholder` versus `<IonLabel position="floating">`?